lib/web_ui/lib/src/engine/configuration.dart

```diff
@@ -249,6 +278,7 @@ class JsFlutterConfiguration {}
 
 extension JsFlutterConfigurationExtension on JsFlutterConfiguration {
   external String? get canvasKitBaseUrl;
+  external String? get canvasKitVariant;
```

Comment:
This is going to need some documentation.

Will this hit stable at the same time as https://github.com/flutter/flutter/issues/114708 ? I am going to have some time to add to the doc site.

Reply:
I'm getting close to landing this, yes. Happy to work with you on the documentation.